.github/workflows/ci.yml

```diff
@@ -24,21 +24,18 @@ jobs:
       matrix:
         include:
           # Test with all supported Django versions, for all compatible Python versions.
-          # See https://docs.djangoproject.com/en/4.0/faq/install/#what-python-version-can-i-use-with-django for the official matrix.
+          # See https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django for the official matrix.
           # Additionally test on Django’s main branch with the most recent Python version.
           - python: "3.8"
             toxenv: py38-dj32,py38-dj41,py38-dj42
           - python: "3.9"
             toxenv: py39-dj32,py39-dj41,py39-dj42
           - python: "3.10"
-            # Skip testing Django 4.0, already tested in previous workflow job.
             toxenv: py310-dj32,py310-dj41,py310-dj42,py310-djmain
           - python: "3.11"
             toxenv: py311-dj41,py311-dj42,py311-djmain
```

Comment:
could you add dj50 to the mix? and drop 4.1 (since it is EOL https://endoflife.date/django)